lacework/go-sdk
# feat(cli): Select query language only on query creation

## Summary

For lacework query create command, if a customer doesn’t run it with a -f/--file or -u/--url option, it will be in EDITOR mode and a template query text will be in a editor for the customer to edit.

Rego query needs a different query text template than LQL, so we need to add a selector to it so it’s known which query template to provide.  The supported language can be either `Rego` or `LQL`. By default it's `LQL`. This selector will be only displayed to customers for whom Rego is enabled.  

## How did you test this change?
* build and install the CLI from source:  `make prepare && make install-cli`
* Run query create `lacework query create`, it enters interactive mode and the survey works as expected.
